README.md

```diff
@@ -54,6 +54,65 @@ demo.rb -- 8 warnings:
   [3]:Dirty#awful has unused parameter 'y' (UnusedParameters)
 ```
 
+## Fixing Smell Warnings
+
+Since `reek` mostly focuses on high-level code smells we can't tell you how to fix warnings in
+a generic fashion since this is and will always be completely dependent on your domain
+language and bussiness logic.
+
+A simple example though might help you get going. Have a look at the sample code below using a
+"Ruby on Rails" model (be aware though that this example is truncated and not really working code):
```

Comment:
It might be the academic in me, but ‘mostly’, ‘simple’ and ‘really’ smell to me as weasel words; also, ‘since’ and ‘though’ are repeated in the same sentence. How about:

```
`reek` focuses on high-level code smells, so we can't tell you how to fix warnings in
a generic fashion; this is and will always be completely dependent on your domain
language and bussiness logic.

That said, an example might help you get going. Have a look at this sample of a
Ruby on Rails model (be aware that this example is truncated and not working code):
```